src/sentry/utils/snuba.py

```diff
@@ -977,19 +981,7 @@ def _bulk_snuba_query(
             raise UnexpectedResponseError(f"Could not decode JSON response: {response.data!r}")
 
         if response.status != 200:
-            error_query = snuba_param_list[index][0]
-            if isinstance(error_query, Request):
-                query_str = error_query.serialize()
-                query_type = "mql" if isinstance(error_query.query, MetricsQuery) else "snql"
-            else:
-                query_str = json.dumps(error_query)
-                query_type = "snql"
-            sentry_sdk.add_breadcrumb(
-                category="query_info",
-                level="info",
-                message=f"{query_type}_query",
-                data={query_type: query_str},
-            )
+            _log_request_query(snuba_param_list[index][0])
```

Comment:
this is a piece of code I wrote myself a few days ago, I extracted it to its own method. I was able to remove 1 of the type checks bc of a recent type signature change for _bulk_snuba_query